drug-interaction/README.md

```diff
@@ -0,0 +1,41 @@
+### Drug-drug interaction
+
+
+## Problem
+In this example, we attempt to infer unknown DDIs from a network of
+multiple drug-based similarities and known interactions. 
+
+## Dataset
+Each dataset below  contain seven drug–drug similarities. Four of these
+similarity measures are drug-based: Chemical-based, Ligand-based,
+Side-effect-based and Annotation-based. Three similarities are be-
+tween drug targets and computed by aggregating over known targets
+for the drugs: Sequence-based, PPI network-based, and Gene
+Ontology-based. 
+
+- general-interactions: This dataset came from DrugBank version 4.3 for a total of 4293 interactiosn across 315 drugs.  There is also a file DrugBankIDs, which maps integer IDs to DrugIDs.  Each fold share the same similarity data. 
```

Comment:
Use a comma to denote thousand (this is not a general rule, but since you did it for other numbers in this file).